tools/nimbus-gradle-plugin/src/main/groovy/org/mozilla/appservices/tooling/nimbus/NimbusGradlePlugin.groovy

```diff
@@ -213,13 +199,11 @@ class NimbusPlugin implements Plugin<Project> {
             println("Downloading nimbus-fml cli version $asVersion")
 
             def successfulHost = tryDownload(archive.getParentFile(), archive.getName(),
-                    // …the latest one from github.
-                    "https://github.com/mozilla/application-services/releases/download/v$asVersion",
-                    "https://github.com/mozilla/application-services/releases/latest/download"
+                    "https://github.com/mozilla/application-services/releases/download/v$asVersion"
             )
 
             if (successfulHost == null) {
-                throw java.io.IOException("Unable to download nimbus-fml tooling")
+                throw java.io.IOException("Unable to download nimbus-fml tooling. If you are using a development version of the Nimbus Gradle Plugin, please set applicationServicesDir appropriately.")
```

Comment:
Could we maybe describe what the setting of `applicationServicesDir` looks like? is this the same as autopublish flow in fenix, where we set property in the `local.properties` (i.e https://github.com/mozilla-mobile/fenix#auto-publication-workflow-for-android-components-and-application-services)

if so maybe just adding `please set applicationServicesDir in the local.properties appropriately` could help developers quickly understand it's the same thing their used to do

Reply:
Actually it doesn't need to go into local.properties, necessarily, it'll need to be in the nimbus block. I've got a little snippet that helps with it and plan to include it when we move to upgrade a-c/focus/fenix to use this.

Reply:
Awesome!! If the snippet is easy to reference on its own, is it worth putting the snippet in the changelog?

Reply:
Well it's not relevant in all cases, specifically not relevant for a-s modules that might be using this

Reply:
Love the new error!

Reply:
Extreme specificity wins again!